content/tracing/advanced_usage/_index.md

```diff
@@ -1454,12 +1454,37 @@ If logs are already JSON formatted, there should be nothing left to do.
 [2]: https://docs.datadoghq.com/logs/log_collection/java/?tab=log4j#configure-your-logger
 {{% /tab %}}
 {{% tab "Python" %}}
-Getting the required information needed for logging is easy:
 
-```python
-from ddtrace import helpers
+To inject trace information from Python into logs:
+
+1. Set the environment variable `DD_LOGS_INJECTION=true` when using `ddtrace-run` or manually patch the `logging` module.
+2. Update your log formatter to include ``dd.trace_id`` and ``dd.span_id`` attributes from the log record.
+
+This integration for logs works if the log entry includes:
+
+- `dd.trace_id=%(dd.trace_id)s`
+- `dd.span_id=%(dd.span_id)s`
+
+An example of this in practice with manual instrumentation:
```

Comment:
Do we have a `ddtrace-run` example we can put here first?

People will likely just copy/pasta what you have here to try and make it work, we'd like to encourage `ddtrace-run` first, manual patching as the backup.

Reply:
Added `ddtrace-run` example